napari/utils/perf/_patcher.py

```diff
@@ -183,18 +184,18 @@ def my_announcer(wrapped, instance, args, kwargs):
     for target_str in callables:
         if target_str in patched:
             # Ignore duplicated targets in the config file.
-            print(f'Patcher: [WARN] skipping duplicate {target_str}')
+            logging.warning('skipping duplicate %s', target_str)
             continue
 
         # Patch the target and note that we did.
         try:
             module, attribute_str = _import_module(target_str)
             if module is not None and attribute_str is not None:
                 _patch_attribute(module, attribute_str, patch_func)
-        except PatchError as exc:
+        except PatchError:
             # We don't stop on error because if you switch around branches
             # but keep the same config file, it's easy for your config
             # file to contain targets that aren't in the code.
-            print(f'Patcher: [ERROR] {exc}')
+            logging.exception('Something went wrong while patching')
```

Comment:
Should we not log the exception message itself?

Reply:
`logging.exception` logs the exception without passing it explicitly by doing some magic inspection of the stack trace!

Reply:
🤯 

Can you add a comment above it pointing that out? Cos that is just batshit crazy. 😂